gems/pending/spec/miq_vm/miq_azure_vm_instance_spec.rb

```diff
@@ -12,8 +12,8 @@
     @client_id               = @test_env[:azure_client_id]
     @client_key              = @test_env[:azure_client_key]
     @tenant_id               = @test_env[:azure_tenant_id]
-    @instance_name           = @test_env[:instance_name]
-    @instance_resource_group = @test_env[:instance_resource_group]
+    @instance_name           = "miq-test-rhel1"
+    @instance_resource_group = "miq-azure-test1" 
 
```

Comment:
@bronaghs this shouldn't change, it's handled automatically by the `TestEnvHelper`